src/main/java/com/deavensoft/paketomat/center/CenterController.java

```diff
@@ -0,0 +1,46 @@
+package com.deavensoft.paketomat.center;
+
+import com.deavensoft.paketomat.center.model.Package;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("api/packages")
+public class CenterController {
+    private CenterServiceImpl centerServiceImpl;
+    @Autowired
+    public CenterController(CenterServiceImpl centerServiceImpl)
+    {
+        this.centerServiceImpl = centerServiceImpl;
+    }
+
+    @GetMapping
+    public List<Package> getAllPackages()
+    {
+        return centerServiceImpl.getAllPackages();
+    }
+
+    @PostMapping
+    public int savePackage(@RequestBody Package newPackage)
+    {
+        centerServiceImpl.save(newPackage);
+        return 1;
+    }
+    @GetMapping(path = "{id}")
```

Comment:
The path should look like api/packages/1.

- [ ] Add / before {id}
- [ ] Do the same change for all getById methods in the other controllers